spec/components/show_embargo_component_spec.rb

```diff
@@ -39,6 +53,7 @@
   end
 
   context 'not embargoed with release date' do
+    let(:allows_modification) { true }
     let(:document) do
       SolrDocument.new(
         id: 1,
```

Comment:
same deal as above ... new context blocks so we can test locked/unlocked object and edit icon status ... indenting causes a big diff